core/src/main/scala/akka/persistence/cassandra/snapshot/CassandraSnapshotStatements.scala

```diff
@@ -79,6 +86,19 @@ import akka.persistence.cassandra.FutureDone
         ${limit.map(l => s"LIMIT ${l}").getOrElse("")}
     """
 
+  def selectLatestSnapshotMeta =
+    s"""SELECT persistence_id, sequence_nr, timestamp FROM ${tableName} WHERE
+    persistence_id = ? 
+    ORDER BY sequence_nr DESC
+    LIMIT ?
+    """
+
+  def selectAllSnapshotMeta =
+    s"""SELECT sequence_nr, timestamp FROM ${tableName} WHERE
+    persistence_id = ? 
+    ORDER BY sequence_nr DESC
+    """
```

Comment:
can we use the existing `selectSnapshotMetadata` instead of adding two more?
the table is ordered by sequence_nr DESC

Reply:
I think we need them. 
The first as we want to decide the LIMIT per query rather than fixed. 
The second as we don't want to provide a sequence nr. We could use 0 and Long.MAX but seems better to let C* know we're not filtering on sequence_nr